forc-util/Cargo.toml

```diff
@@ -16,8 +16,10 @@ clap = { workspace = true, features = ["cargo", "derive", "env"] }
 dirs.workspace = true
 fd-lock.workspace = true
 forc-tracing.workspace = true
+fuel-abi-types.workspace = true
 fuel-asm.workspace = true
 fuel-tx = { workspace = true, optional = true }
+fuels-core.workspace = true
```

Comment:
this should also be optional

Reply:
done in https://github.com/FuelLabs/sway/pull/6968/commits/ee4d69d32b052f37256fb9da10dceba59874d6e8